### PR TITLE
Fix order of precedence list

### PR DESCRIPTION
**Pull request by ulyssessouza**
Fix precedence list

### CODE REVIEW

File: compose/envvars-precedence.md [hunk 1]
```diff
@@ -21,30 +18,32 @@ Each table's:
...
-* `compose.yaml`:`environment` key - In `environment` key from the service section in the `compose.yaml` file.
 * `run --env` - environment variable set via the command line: `docker compose run -e <KEY[=[VAL]]>`.
+* `compose.yaml`:`environment` key - In `environment` key from the service section in the `compose.yaml` file.
+* `compose.yaml`:`env_file` key - In `env_file` key from the service section in the `compose.yaml` file.
+* Image `ENV` - `ENV` directive in the Dockerfile for the image.
+* `Host OS` environment - the environment where the Docker Engine is running.
+* `.env` file - the `.env` file on the project root.
 * RESOLVED AS - This column expresses the result available in the container. So this column expresses the value of `TAG`, our example in this table, after Compose applies the precedence rule.
+
+Obs: The columns "`Host OS` environment" and "`.env` file" are listed only for lookup. These columns cannot result in a variable in the container by itself.
```
Comment (glours) [inline compose/envvars-precedence.md:49]:
```suggestion
> Note:
> Obs: The columns "`Host OS` environment" and "`.env` file" are listed only for lookup. These columns cannot result in a variable in the container by itself.
```

**Review by glours — DISMISSED** [review]
LGTM
A small nit to highlight the observation at the bottom of the file

File: compose/envvars-precedence.md (hunk 1 above)
Comment (usha-mandya) [reply compose/envvars-precedence.md:49]: Suggest we also remove "Obs"

**Review by YashasviChaurasia — DISMISSED** [review]

**Review by laurazard — APPROVED** [review]
LGTM

**Review by StefanScherer — APPROVED** [review]
LGTM

**Comment by usha-mandya** [comment]
Thanks @ulyssessouza. Merging this PR. @aevesdocker will suggest edits in a follow-up PR to improve this page.

**Pull request merged**